exercises/sgf-parsing/canonical-data.json

```
@@ -170,6 +170,185 @@
+            },
+            "expected": {
+                "properties": {
+                    "A": ["\\"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "dc13ca67-fac0-4b65-b3fe-c584d6a2c523",
+            "description": "Opening bracket within property value doesn't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[x[y\\]z][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["x[y]z", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "a780b97e-8dbb-474e-8f7e-4031902190e8",
+            "description": "Semicolon in property value doesn't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[a;b][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["a;b", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "0b57a79e-8d89-49e5-82b6-2eaaa6b88ed7",
+            "description": "Parentheses in property value don't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[x(y)z][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["x(y)z", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "c72a33af-9e04-4cc5-9890-1b92262813ac",
+            "description": "Escaped tab in property value is converted to space",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[hello\\\tworld])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["hello world"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "3a1023d2-7484-4498-8d73-3666bb386e81",
+            "description": "Escaped newline in property value is converted to nothing at all",
```

For consistency with the other test cases:

```suggestion
            "description": "escaped newline in property value is converted to nothing at all",
```

---

Would it perhaps make sense to have this test case moved up above the previous test case, as that test case also depends on escaped newlines being converted to nothing at all?

---

For this particular case (3a1023d2-7484-4498-8d73-3666bb386e81 "escaped newline in property value is converted to nothing at all", this is the first case I see in this file that has an escaped newline) - am I thinking of a different case than the one you were talking about?

---

Ignore this comment. For some reason I thought that there was another test case using escaped newlines. Sorry.